### DISCUSSION TITLE
CMake config files issue?

### DISCUSSION THREAD

**Discussion by loryruta** [microsoft/onnxruntime#22981] [Release Q&A]
I'm trying to depend on onnxruntime from my C++ project and to do that, I've downloaded the latest release:

[onnxruntime-linux-x64-gpu-1.20.1.tgz](https://github.com/microsoft/onnxruntime/releases/download/v1.20.1/onnxruntime-linux-x64-gpu-1.20.1.tgz)

Then from my cmake:

```cmake
set(onnxruntime_DIR "${CMAKE_CURRENT_SOURCE_DIR}/onnxruntime/lib64/cmake/onnxruntime")
find_package(onnxruntime REQUIRED CONFIG)
```

First issue:
- `libonnxruntime.so.1.20.1` is searched inside lib64 instead of lib (the former doesn't exist)
-  `INTERFACE_INCLUDE_DIRECTORIES` is set to `include/onnxruntime` while it should be `include`

**Comment by fs-eire** [comment]
Thank you for reporting. could you please help to create a PR for a fix?

**Comment by sandinomorales** [comment]
Is there a fix for this issue already?
I found the same problem for version 1.22.2.

It seems that the path for the includes is also wrong. In onnxruntimeTargets.cmake, the path for the includes is defined as:
```
set_target_properties(onnxruntime::onnxruntime PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include/onnxruntime"
)
```
Which give me this  error:
```
CMake Error in CMakeLists.txt:
  Imported target "onnxruntime::onnxruntime" includes non-existent path

    "/opt/onnxruntime/onnxruntime-linux-x64-1.22.0/include/onnxruntime"
```

 Deleting the tailing `onnxruntime` fixed it

**Comment by snnn** [comment]
It is because originally when we published this thing the first time, Ubuntu didn't use "/lib64" as the lib folder for 64-bit systems. They used "/lib".  Now if we change it, a lot of projects' build scripts will break, including https://github.com/microsoft/onnxruntime-extensions, https://github.com/microsoft/onnxruntime-genai and many our customers.
